docs/content-translation-process.md

```diff
@@ -7,10 +7,10 @@ As one of the main missions of the PlanB Network Content repo is to lower down t
 If you have another approach that seems relevant, feel free to post an [issue](https://github.com/DecouvreBitcoin/sovereign-university-data/issues/new) to discuss about it. 
 
```

Comment:
@Asi0Flammeus how can we discuss the approach here?
Maybe we can delete these two paragraphs until we have more relevant info tu publish?